python/test/pipeline_test.py

```diff
@@ -450,10 +451,10 @@ def testStartCountdown(self):
   def testStartEta(self):
     """Tests starting a pipeline with an eta."""
     stage = NothingPipeline('one', 'two', three='red', four=1234)
-    eta = datetime.datetime.utcnow() + datetime.timedelta(seconds=30)
+    eta = datetime.datetime.utcnow().replace(tzinfo=pytz.UTC) + datetime.timedelta(seconds=30)
```

Comment:
Unless it is proved that pytz is bundled somehow the test could convert now() to utc (for the comparing the result from taskqueue using code as suggested http://stackoverflow.com/questions/4563272/how-to-convert-a-python-utc-datetime-to-a-local-datetime-using-only-python-stand)